matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/algorithms/megolm/MXOutboundSessionInfo.kt

```diff
@@ -23,9 +23,8 @@ import timber.log.Timber
 internal class MXOutboundSessionInfo(
         // The id of the session
         val sessionId: String,
-        val sharedWithHelper: SharedWithHelper) {
-    // When the session was created
```

Comment:
Why removing this comment? This is not true anymore?

Reply:
reverted